Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.21.3-alpine3.18
+
+COPY . /app
+WORKDIR /app
+
+RUN mkdir /app/bin
+RUN mkdir /app/public
```

Comment:
if I wanted to persist the uploaded files between restarts, would i have to mount a volume here? maybe a `VOLUME` directive would be a good idea here

Reply:
@boudra  you should be able to do it anyway when you start the container right?
but I didn't set it as our standard way to do it just because we don't have a way to persist the full state of the system including the blockchain state until anvil doesn't dump events https://github.com/foundry-rs/foundry/issues/5906